autoinstall/Windows/win_server/bios/Autounattend.xml

```diff
@@ -139,5 +145,5 @@
             </FirstLogonCommands>
         </component>
     </settings>
-    <cpi:offlineImage cpi:source="catalog://desktop-pumkr1e/users/test/downloads/share/0621/install_windows server 2019 serverdatacenter.clg" xmlns:cpi="urn:schemas-microsoft-com:cpi" />
+    <cpi:offlineImage cpi:source="wim:C:/install.wim#Windows Server 2025 SERVERDATACENTER" xmlns:cpi="urn:schemas-microsoft-com:cpi" />

```

Comment:
Does this also work for Windows Server 2022?

Reply:
Yes.